docs/api.md

```diff
@@ -935,9 +938,22 @@ saturation of 5.0. Eating food increases the saturation as well as the food bar.
 Edit these numbers to tweak gravity, jump speed, terminal velocity, etc.
 Do this at your own risk.
 
+#### bot.time.doDaylightCycle
+
+Whether or not the gamerule doDaylightCycle is true or false.
+
+#### bot.time.bigTime
+
+The total number of ticks since day 0.
+
+This value is of type BigInt and is accurate even at very large values. (more than 2^51 - 1 ticks)
+
 #### bot.time.time
 
-Total time of the world since day 0.
+The total numbers of ticks since day 0.
+
+Because the Number limit of Javascript is at 2^51 - 1 bot.time.time becomes inaccurate higher than this limit and the use of bot.time.bigTime is recommended.  
+Realistically though you'll probably never need to use bot.time.bigTime as it will only reach 2^51 - 1 ticks naturally after ~14280821 real years.  
```

Comment:
so why is bigTime exposed then ?

Reply:
My thought was that in a normal situation you'd probably never need it.  
But if someone was working with some highly modified server that messed with the time a lot, it might reach the point where it becomes inaccurate.  
So I guess I want to give to option if someone ever needs it?